tmpl/partials/footer.html

```diff
@@ -52,7 +52,7 @@
       <hr class="ipw-sectioner" style="margin-bottom: 15px;" />
       <div class="col-xs-12">
         <div class="ipw-copyright">
-          &copy; Interplanetary Networks, Inc. |
+          &copy; IPFS Community |
```

Comment:
Every contributor owns the copyright for their respective commits, from what I understand.